### ISSUE TITLE
Freezes on binary data

### ISSUE THREAD

**Issue by davidrenne** [shuLhan/go-bindata#34]
I added a binary file of 800KB approximately to my go-bindata directory and it froze up the command and does not exit.

[Archive.zip](https://github.com/shuLhan/go-bindata/files/3524882/Archive.zip)

Any ideas on why?

I included the simple source of the autohotkey script I built to compile the EXE for windows.  My environment is a mac.  I also tried adding the Archive.zip and that fails too. 

Is binary not supported?

**Comment by shuLhan** [comment]
> Any ideas on why?

Unfortunately, its works on my machine.  Here is the example,

```
23:20 ~/tmp
master ms 0 % ls -l data
total 816
-rwx------ 1 ms ms    338 Aug 21 05:52 Spacebar.ahk
-rwx------ 1 ms ms 827904 Aug 21 05:52 Spacebar.exe
```
and here is the output after running `go-bindata data/`

[bindata.go.txt](https://github.com/shuLhan/go-bindata/files/3526242/bindata.go.txt)

> Is binary not supported?

Binary is supported, there is no exception except that if you have "-ignore" option defined on CLI.

To debug this, you may add files one by one and see which file is causing the issue, and maybe its not the file probably other things (disk full, for example).

**Comment by davidrenne** [comment]
Yea I think my issue was some kind of recursive issue.

`go-bindata -o assets/assets.go assets/...`

This was my command and although I deleted assets.go before each run, I did notice it growing to very large sizes as if it was compiling itself into itself and growing exponentially.

Thats probably why it got stuck.  If you know of a way to not compile yourself into yourself, that could be a bug fix, but I will close this issue for now.  No big deal.
